content/news/cfp-close.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Thank you ❤️ PyLadiesCon 2024 received many proposals!"
+meta_title: "Call for Proposal of PyLadiesCon 2024 was closed and we have received almost 200 proposals!"
+description: "Call for Proposal of PyLadiesCon 2024 was closed on 15th September and we have received almost 200 proposaled! Thank you for all that has been submitted, in our minds you are super awesome and we would love to see your talk. However, the reviews will have a busy weeks ahead to review all the proposals and we will have tough decisions to make regarding which talks to accept. We aim to publish a tentative schedule by 8th October.
+"
+date: 2024-09-17T05:00:00Z
+categories: ["Blog Post",]
+author: "Cheuk"
+tags: ["python", "community", "conference", "public speaking", "pyladies"]
+draft: false
+layout: news
+---
+
+Call for Proposal of PyLadiesCon 2024 was closed on 15th September and we have received almost 200 proposaled! Thank you for all that has been submitted, in our minds you are super awesome and we would love to see your talk. However, the reviews will have a busy weeks ahead to review all the proposals and we will have tough decisions to make regarding which talks to accept. We aim to publish a tentative schedule by 8th October.
+
+### So many proposals!
+
+This year we accept talks in multiple languages and we have received in total 192 proposals, an increase from last year's 164. Here are the number of proposals that we received in different languages:
+
+- English: 147
+- Español: 31
+- Português: 12
+- Deutsch: 0
+- 日本語: 2
+- 中文: 0
+
+Despite the effort to promote more diversity languages and the support of many PyLadies Chapters, there are still not a lot of submissions besides English and Español. In the future, more strategic planning regarding which language we would like to invite folks to submit would be needed to plan ahead. Nevertheless, a big thanks to the following PyLadies Chapters for helping us during the Call for Proposal periods:
+
+- PyLadies Tokyo
+- PyLadies Berlin
+- PyLadies Taiwan
+- PyLadies Maputo
+- PyLadies Colombia
+- PyLadies Santo Domingo
+- PyLadies São Carlos
+
```

Comment:
Can we also mention thanks to people who helped by sharing and boosting our CFP?
```suggestion

And thank you to everyone who supported our CfP by sharing our news and posts on social media. It is with your help that we're able to break this record.
```